Cloudinary-pwa-kit/packages/pwa-cloudinary/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "pwa-cloudinary",
```

Comment:
what is the impact of this change?

Reply:
In MRT the project name was "cloudinary". As per the documentation the project name in MRT and the package name in package.json should be some, otherwise, the package will not upload on MRT.